arbnode/batch_poster.go

```diff
@@ -1246,6 +1249,22 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 		b.building.msgCount++
 	}
 
+	firstMsgTimeStamp := firstMsg.Message.Header.Timestamp
+	firstMsgBlockNumber := firstMsg.Message.Header.BlockNumber
+	batchNearL1BoundMinTimestamp := firstMsgTimeStamp <= l1BoundMinTimestamp+uint64(config.ReorgResistanceMargin/time.Second)
+	batchNearL1BoundMinBlockNumber := firstMsgBlockNumber <= l1BoundMinBlockNumber+uint64(config.ReorgResistanceMargin/ethPosBlockTime)
```

Comment:
I think `l1BoundMinTimestamp` and `l1BoundMinBlockNumber` might not be filled in here. In particular, I think you need `hasL1Bound && config.L1BlockBoundBypass > 0`

Even then, they seem to already be padded by `config.L1BlockBoundBypass`. I think we might just need a new variable that's unrelated to the L1BlockBoundBypass config, and we should rename this existing variable to clarify it's source.

Reply:
I agree, fixed it.